charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -7,6 +7,8 @@ java:
     S2S_NAME: "bulk_scan_payment_processor"
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     PAY_HUB_URL: http://ccpay-bulkscanning-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    IDAM_API_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
+    IDAM_CLIENT_REDIRECT_URI: https://bulk-scan-orchestrator-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2/callback
```

Comment:
These envs would need to overrided in preview yaml to use AAT ones ?

Reply:
So `{{ .Values.global.environment }}` won't resolve to `aat` in aat?